docker-compose.yml

```
@@ -0,0 +1,38 @@
+    environment:
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_DASHBOARDS_JSON_ENABLED=true
+    volumes:
+      - ./grafana-datasources.yml:/etc/grafana/provisioning/datasources/datasources.yml
+      - ./dashboards:/var/lib/grafana/dashboards
+      - ./provisioning:/etc/grafana/provisioning
+
+volumes:
+  prometheus_data:
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)